roles/pulp_common/defaults/main.yml

```diff
@@ -44,7 +44,7 @@ epel_release_packages:
 pulp_irregularly_named_plugins:
   - galaxy-ng
 pulp_pkg_pulpcore_name: "{{ pulp_pkg_name_prefix }}pulpcore"
-pulp_pkg_repo:
+pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.16/el{{ ansible_distribution_major_version }}/$basearch/"
```

Comment:
My only comment would be a usability one. If a user wanted a different version, they would have to construct the whole URL themselves to pass in. If you had a variable to represent version though, you could include that in this string and allow ease of use for users since the URL structure is consistent.

Reply:
@ehelms Yeah, I see what you mean.

However, often the logic (or other default / OS-specific variables) in the installer are specific to the specified version of the repo. Like the package names, upgrade logic, etc. Letting a users specify a specific version easily would imply that they can use another version even when they cannot reliably do so.

And installing from packages is still an advanced use case.

So I am leaning against it adding a variable, thus making the variables list longer.